.gitignore

```diff
@@ -0,0 +1,2 @@
+
+.DS_Store
```

Comment:
Remove this or just `git rm .gitignore` for now.  I don't like this sort of thing in the repo ignore.  This should be in your global .gitignore (not the repository .gitignore).

Reply:
You'll need to google how to set this up on mac because it may not be standard.